test/db/analysis/x86_64

```diff
@@ -3391,38 +3391,109 @@ f~case.
 EOF
 EXPECT=<<EOF
 0x140001023 1 switch.0x140001023
+0x140001f5b 1 switch.0x140001f5b
```

Comment:
Maybe not for this PR, but this is really weird... There is a switch table here but there is no function defined. However at 0x140001e38 you can see `jbe   0x140001f40`, which goes just there. It's strange no basicblock was created there.